portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,6 +32,12 @@
 public class DataServlet extends HttpServlet {
 
   DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+  ImagesService imagesService = ImagesServiceFactory.getImagesService();
+
+  List<String> validFileTypes = new ArrayList<>(
```

Comment:
You can just do `Arrays.asList` without the enclosing `new ArrayList`

Reply:
Thanks for pointing that out! I might have accidentally added it and missed it when reviewing my code.